esmvaltool/diag_scripts/shared/_base.py

```diff
@@ -443,28 +441,38 @@ def main(cfg):
     logger.info("Starting diagnostic script %s with configuration:\n%s",
                 cfg['script'], yaml.safe_dump(cfg))
 
-    # Create output directories
+    # Clean run_dir and output directories from previous runs
+    default_files = ['log.txt', 'resource_usage.txt', 'settings.yml']
```

Comment:
```suggestion
    default_files = {'log.txt', 'profile.bin', 'resource_usage.txt', 'settings.yml'}
```

Reply:
There is a fourth file that can be generated, if `profile_diagnostic` is set in config-user.yml